cumulusci/tasks/salesforce/CreateCommunity.py

```diff
@@ -119,5 +120,5 @@ def _poll_action(self):
 
     def _get_community(self):
         community_list = self.sf.restful("connect/communities")["communities"]
-        communities = {c["name"]: c for c in community_list}
-        return communities.get(self.options["name"])
+        communities = {c["name"].lower().upper(): c for c in community_list}
```

Comment:
Was going to ask why this needed but then read the conversation thread. TIL.

Reply:
Yeah, I went all the way down the rabbit hole on this one.